### PR TITLE
Update communication links

### LINKED ISSUE #1: Decide on a team chat, discussion, organizing place
* IRC
* Discord
* Somewhere else?

### PR DESCRIPTION
**Pull request by kfv**
Closes #1

### CODE REVIEW

File: howwework.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # How We Work
 ## Communications
-- Real-time messaging: [BSDLabs Discord server](https://discord.gg/9vZzzxz7y9)
-- Persisted documentation: Primarily on GitHub bsdlabs/ until we need/ find a better place.
```
Comment (kfv) [inline howwework.md:4]: Rephrased to keep it simple for any new member/observer as we haven't thought of any change yet - we'd update it if required anytime.

**Comment by kfv** [comment]
By the way, I think we can change the filename to something shorter, like \`structure' or \`procedures', I don't know. Or at least we can hyphenate the current name so it reads better. Let me know what you think, please.

**Review by koobs — APPROVED** [review]
Looks good, thank you @kfv :)

**Comment by koobs** [comment]
> By the way, I think we can change the filename to something shorter, like \`structure' or \`procedures', I don't know. Or at least we can hyphenate the current name so it reads better. Let me know what you think, please.

playbook.md :)

This can be a separate change

**Comment by kfv** [comment]
Roger that, I'll do another commit to rename the file.

**Pull request merged**